configure.ac

```diff
@@ -488,33 +488,34 @@ AS_IF([test x$have_gui = xyes],
   # settings for Qt
   LIBS="$LIBS $QT_LIBS"
   PKG_FLAGS="$PKG_FLAGS $QT_CFLAGS"
+  AC_MSG_CHECKING([Do we have to add -fPIC flag to make Qt happy])
```

Comment:
This leads to a strange sentence displayed during `./configure`:

```
checking Do we have to add -fPIC flag to make Qt happy...
```

That's why I suggested:

```
AC_MSG_CHECKING([if we have to add -fPIC to make Qt happy])
```

I must confess that I made the same mistake before, see #65.